rust/tool/tests/common/mod.rs

```diff
@@ -17,6 +17,24 @@ use rand::{thread_rng, Rng};
 use serde_json::json;
 use sha2::{Digest, Sha256};
 
+use lanzaboote_tool::esp::Architecture;
+
+/// Returns the host platform system
+/// in the system double format for
+/// our usual targets.
+#[cfg(target_arch="aarch64")]
```

Comment:
This should be a conditionally compiled constant instead of a function.

Reply:
Done